blocks.md

```diff
@@ -384,40 +396,47 @@ Creates a block which displays disk space information.
 [[block]]
 block = "disk_space"
 path = "/"
-alias = "/"
 info_type = "used"
 unit = "GiB"
-format = "{icon}{used}/{total} {unit} ({available}{unit} free)"
+format = "{icon} {used}/{total} ({available} free)"
```

Comment:
Is the space added because the icon no longer has spaces inside of it?

Reply:
Yes